front/tests/012-op-simulation-settings-tab.spec.ts

```diff
@@ -220,7 +257,21 @@ test.describe('Simulation Settings Tab Verification', () => {
     await operationalStudiesPage.returnSimulationResult();
     await opTimetablePage.getTrainArrivalTime('12:03');
     await opTimetablePage.clickOnScenarioCollapseButton();
+    // TODO: Remove the waitForTimeout and firefox from performOnSpecificOSAndBrowser when issue #9909 is fixed
+    await page.waitForTimeout(stabilityTimeout);
```

Comment:
Hm, now that the root cause has been fixed, this can be removed right?

Reply:
Oops, I forgot to remove them after the `autosquash `

Reply:
Removed